lutris/util/vulkan.py

```diff
@@ -19,12 +21,24 @@ def search_for_file(directory):
     return False
 
 def vulkan_check():
-    vulkan_lib = search_for_file("/usr/lib")
-    vulkan_lib32 = search_for_file("/usr/lib32")
-    vulkan_lib_multi = search_for_file("/usr/lib/x86_64-linux-gnu")
-    vulkan_lib32_multi =  search_for_file("/usr/lib32/i386-linux-gnu")
-    has_32_bit = vulkan_lib32 or vulkan_lib32_multi
-    has_64_bit = vulkan_lib or vulkan_lib_multi
+    has_32_bit = False
+    has_64_bit = False
+    pattern = re.compile(r'^libvulkan\.so')
+    proc = subprocess.Popen("ldconfig -p", shell=True, stdout=subprocess.PIPE)
+    for line in io.TextIOWrapper(proc.stdout, encoding="utf-8"):
+        if line.find('libvulkan') != -1:
+            if line.find('x86-64') != -1:
+                has_32_bit = True
+            else:
+                has_64_bit = True
+
+    if not (has_32_bit or has_64_bit):
+        vulkan_lib = search_for_file("/usr/lib")
```

Comment:
I find these fallbacks questionable. If `ldconfig` didn't find vulkan than no application will, so its pointless to find files otherwise.